frontend/src/modules/auth/store/auth.getters.ts

```diff
@@ -0,0 +1,18 @@
+import { AuthState } from '@/modules/auth/store/auth.state';
+import { AuthService } from '@/modules/auth/services/auth.service';
+import { LfRole } from '@/shared/modules/permissions/types/Roles';
+import { TenantUser } from '@/modules/auth/types/TenantUser.type';
+
+export default {
+  tenantUser: (state: AuthState): TenantUser => {
+    const tenantId = AuthService.getTenantId();
+    return (state.user?.tenants || []).find((t) => t.tenantId === tenantId);
+  },
+  roles: (state: AuthState): LfRole[] => state.tenantUser.roles || [LfRole.readonly],
```

Comment:
In Manage projects page I'm getting "Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'roles')"